core/domain/wipeout_service_test.py

```diff
@@ -589,7 +603,29 @@ def test_run_user_deletion_completion_with_user_wrongly_deleted(self):
             expected_args=[('WIPEOUT: Account deletion failed', email_content)]
         )
 
-        with send_email_swap:
+        with send_email_swap, self.swap(feconf, 'CAN_SEND_EMAILS', True):
+            self.assertEqual(
+                wipeout_service.run_user_deletion_completion(
+                    self.pending_deletion_request),
+                wipeout_domain.USER_VERIFICATION_FAILURE)
+
+        self.assertIsNotNone(
+            user_models.UserSettingsModel.get_by_id(self.user_1_id))
+        self.assertIsNotNone(
+            auth_models.UserAuthDetailsModel.get_by_id(self.user_1_id))
+        self.assertIsNotNone(
+            user_models.PendingDeletionRequestModel.get_by_id(self.user_1_id))
+
+    def test_run_user_deletion_completion_user_wrongly_deleted_emails_disabled(
```

Comment:
Does this test verify that the emails weren't sent? Ditto above for ensuring they were sent. Both tests don't seem to verify that aspect of the implementation.

Reply:
Ah, yes, forgot about that, fixed.